src/makeCalculator.js

```diff
@@ -37,7 +37,31 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add: (input) => calculator.result + input,
+
+    subtract: (input) => calculator.result - input,
+
+    multiply: (input) => calculator.result * input,
+
+    divide: (input) => calculator.result / input,
```

Comment:
The 'divide' method should modify the 'result' property instead of just returning the division. You may want to refactor it to 'calculator.result /= input'. Also, don't forget to handle the case when 'input' is zero to avoid division by zero error.